Shell/Options.cpp

```diff
@@ -1482,6 +1482,12 @@ void Options::init()
     _inductionOnComplexTerms.onlyUsefulWith(_induction.is(notEqual(Induction::NONE)));
     _lookup.insert(&_inductionOnComplexTerms);
 
+    _inductionOnlyGround = BoolOptionValue("induction_only_ground","indog",true);
+    _inductionOnlyGround.description = "Apply induction only on ground literals vs. literals with at most one free variable";
+    _inductionOnlyGround.tag(OptionTag::INDUCTION);
```

Comment:
What's *in*, dog?

It's a matter of taste, but I might prefer something like `induction_applies_to` with values `ground`, `one_var`. Then with the inevitable `two_var` extension it's just one more enumeration value. No strong preference either way.

Reply:
I am wondering if there is much value in having the option so granular, allowing to choose between `one_var`, `two_vars`, `all_vars` etc. @mezpusz any opinions?

Anyway I don't have a strong preference (just thought of keeping it simple with a boolean).
I can also change it to `induction_ground_only` for the enthusiastic `indgo` :D (and to have it in line with `question_answering_ground_only`)

Reply:
I prefer `indgo`! If we manage to generalize this to any number of free variables, it still wouldn't make sense to make it more granular, since we shouldn't get fundamentally bigger problems with more variables... I think.

Reply:
Cool - you seem to have thought about this more than me. :-)